src/components/DatetimePicker/DatetimePicker.vue

```diff
@@ -49,6 +51,28 @@ export default {
 		Datepicker
 	},
 
-	inheritAttrs: false
+	inheritAttrs: false,
+
+	computed: {
+		firstDay() {
+			return window.firstDay
+				? window.firstDay
+				: 0 // sunday as default
+		},
+		lang() {
+			// fallback to default in case of unavailable data
+			return {
+				days: window.dayNamesShort
+					? window.dayNamesShort			// provided by nextcloud
```

Comment:
I suggest to document/validate this in/through the eslint plugin and not use the window variable:
https://github.com/nextcloud/eslint-plugin-nextcloud/blob/master/lib/environments/nextcloud.js
Then, it will be easier to track possible deprecation in the future.

Reply:
https://github.com/nextcloud/eslint-plugin/pull/111